src/homework3/Task1.java

```diff
@@ -0,0 +1,29 @@
+package homework3;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Task1 {
+
+    private static final String ALPHABET = "abcdefghijklmnopqrstuvwxyz";
+
+    public static void main(String[] args) {
+        System.out.println(randomStringCollection(7, 6, 15));
+    }
+
+    public static ArrayList<String> randomStringCollection(int numberOfStrings, int minLen, int maxLen) {
```

Comment:
```suggestion
    public static ArrayList<String> getRandomStringCollection(int numberOfStrings, int minLen, int maxLen) {
```
We have `randomStringCollection` method and `randomStringCollection` list, so it may lead to the confusion, it's better to rename either method or list name. And I suggest to rename method, because method should represent an ACTION, that it does - if it returns something - in most cases name is starting with "get" and so on.

Reply:
fixed in <Homework3 fixes>